css-nesting-1/Overview.bs

```diff
@@ -242,6 +308,13 @@ Direct Nesting {#direct}
 		}
 		/* Invalid because & isn't in the first compound selector */
 
+		.foo {
+			color: red;
+			&&.bar { color: blue; }
+		}
+		/* Invalid because the 1st & is no longer 
+		   at the start of a nesting selector */
```

Comment:
No, invalid because the *second* `&` isn't at the start of a *compound* selector.